Cargo.toml

```diff
@@ -225,12 +405,92 @@ anemo-build = { git = "https://github.com/mystenlabs/anemo.git", rev = "0f0ae8d8
+sui-macros = { path = "crates/sui-macros" }
+sui-metric-checker = { path = "crates/sui-metric-checker" }
+sui-move = { path = "crates/sui-move" }
+sui-move-build = { path = "crates/sui-move-build" }
+sui-network = { path = "crates/sui-network" }
+sui-node = { path = "crates/sui-node" }
+sui-open-rpc = { path = "crates/sui-open-rpc" }
+sui-open-rpc-macros = { path = "crates/sui-open-rpc-macros" }
+sui-proc-macros = { path = "crates/sui-proc-macros" }
+sui-protocol-config = { path = "crates/sui-protocol-config" }
+sui-protocol-config-macros = { path = "crates/sui-protocol-config-macros" }
+sui-proxy = { path = "crates/sui-proxy" }
+sui-replay = { path = "crates/sui-replay" }
+sui-rosetta = { path = "crates/sui-rosetta" }
+sui-rpc-loadgen = { path = "crates/sui-rpc-loadgen" }
+sui-sdk = { path = "crates/sui-sdk" }
+sui-simulator = { path = "crates/sui-simulator" }
+sui-snapshot = { path = "crates/sui-snapshot" }
+sui-source-validation = { path = "crates/sui-source-validation" }
+sui-storage = { path = "crates/sui-storage" }
+sui-surfer = { path = "crates/sui-surfer" }
+sui-swarm = { path = "crates/sui-swarm" }
+sui-swarm-config = { path = "crates/sui-swarm-config" }
+sui-telemetry = { path = "crates/sui-telemetry" }
+sui-test-transaction-builder = { path = "crates/sui-test-transaction-builder" }
+sui-test-validator = { path = "crates/sui-test-validator" }
+sui-tls = { path = "crates/sui-tls" }
+sui-tool = { path = "crates/sui-tool" }
+sui-transaction-builder = { path = "crates/sui-transaction-builder" }
+sui-transactional-test-runner = { path = "crates/sui-transactional-test-runner" }
+sui-types = { path = "crates/sui-types" }
+sui-upgrade-compatibility-transactional-tests = { path = "crates/sui-upgrade-compatibility-transactional-tests" }
+sui-verifier-transactional-tests = { path = "crates/sui-verifier-transactional-tests" }
 telemetry-subscribers = { path = "crates/telemetry-subscribers" }
+test-utils = { path = "crates/test-utils" }
+transaction-fuzzer = { path = "crates/transaction-fuzzer" }
 typed-store = { path = "crates/typed-store" }
 typed-store-derive = { path = "crates/typed-store-derive" }
+workspace-hack = { path = "crates/workspace-hack" }
+x = { path = "crates/x" }
+
+narwhal-config = { path = "narwhal/config" }
+narwhal-consensus = { path = "narwhal/consensus" }
+narwhal-crypto = { path = "narwhal/crypto" }
+narwhal-dag = { path = "narwhal/dag" }
+narwhal-examples = { path = "narwhal/examples" }
+narwhal-executor = { path = "narwhal/executor" }
+narwhal-network = { path = "narwhal/network" }
+narwhal-node = { path = "narwhal/node" }
+narwhal-primary = { path = "narwhal/primary" }
+narwhal-storage = { path = "narwhal/storage" }
+narwhal-test-utils = { path = "narwhal/test-utils" }
+narwhal-types = { path = "narwhal/types" }
+narwhal-worker = { path = "narwhal/worker" }
+
+sui-execution = { path = "sui-execution" }
+sui-move-natives = { path = "sui-execution/latest/sui-move-natives", package = "sui-move-natives-latest" }
```

Comment:
ditto, execution-layer for everything below this line (inclusive).  But `sui-execution` can stay here.